internal/client/errors/errors.go

```diff
@@ -41,6 +41,24 @@ func IsNotFoundError(err error) bool {
 	return false
 }
 
+func IsUnauthorizedError(err error) bool {
```

Comment:
can you modify this function to retrun another function which will be invoked in the refreshAuthxCtx

Reply:
This will reduce the error being sent as a parameter to refreshAuthxCtx